Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputBindingPropertiesView.cs

```diff
@@ -102,11 +104,109 @@ protected override void DrawGeneralProperties()
                     }
                 }
 
+                showMatchingControls = EditorGUILayout.Foldout(showMatchingControls, "Matching Controls");
+                // Show the specific controls which match the current path
+                if (showMatchingControls)
+                {
+                    EditorGUI.indentLevel++;
+                    DrawMatchingControlPaths();
+                    EditorGUI.indentLevel--;
+                }
                 // Control scheme matrix.
                 DrawUseInControlSchemes();
             }
         }
 
+        /// <summary>
+        /// Finds all registered control paths implemented by concrete classes which match the current binding path and renders it.
+        /// </summary>
+        private void DrawMatchingControlPaths()
+        {
+            var path = m_ControlPathEditor.pathProperty.stringValue;
+            var deviceLayout = new InternedString(InputControlPath.TryGetDeviceLayout(path));
+            var parsedPath = InputControlPath.Parse(path).ToArray();
+
+            bool matchExists = false;
+            EditorGUILayout.BeginVertical();
+            if (parsedPath.Length == 2)
+            {
+                if (deviceLayout != InputControlPath.Wildcard)
+                {
+                    var rootLayout = EditorInputControlLayoutCache.allLayouts.FirstOrDefault(x => x.isDeviceLayout && !x.isOverride && !x.hideInUI && x.name == deviceLayout);
+                    if (rootLayout != null)
+                    {
+                        for (int i = 0; i < rootLayout.m_Controls.Length; i++)
+                        {
+                            if (InputControlPath.MatchControlComponent(ref parsedPath[1], ref rootLayout.m_Controls[i]))
+                            {
+                                EditorGUILayout.LabelField($"{rootLayout.displayName}/{rootLayout.m_Controls[i].displayName}");
+                                matchExists = true;
+                                continue;
```

Comment:
Can this `continue` be removed? Or is this supposed to be a `break`?

Reply:
I think it should be a break. For a controller layout, there shouldn't be instances where 2 buttons have the same usage right?